tokens/src/core/components/Popover.json

```diff
@@ -2,7 +2,7 @@
   "size": {
     "popover": {
       "border": {
-        "width": { "value": "0" }
+        "width": { "value": "0px", "_comment": "A pixel next to zero is necessary for correct display Popover arrow." }
```

Comment:
I left a comment because we have already removed the pixel unit of measurement for this token 3 times 🙅

Reply:
😮